File: stlsolutions/thewalkingAdam.cpp
#include<iostream>
using namespace std;


int main(int argc, char const *argv[]) {

  int t;
  string s;
  cin >> t;
  while (t--) {
    cin >> s;
    int count = 0;
    for (int i = 0; i < s.length(); i++) {
      if(s[i] == 'D'){
        break;
      }
      count++;
    }
    std::cout << count << '\n';
  }
return 0;

}

File: stlsolutions/thewalkingAdam.py
t = input()
t = int(t)
while t > 0:
    t = t-1
    count = 0
    s = input()
    for x in s:
        if x == 'U':
            count +=1
        if x == 'D':
            break
    print(count)
        
